src/service/navigation.js

```diff
@@ -43,6 +43,9 @@ const VALID_TARGETS = ['_top', '_blank'];
 /** @private @const {string} */
 const ORIG_HREF_ATTRIBUTE = 'data-a4a-orig-href';
 
+/** @const {string} */
+export const AMP_CUSTOM_LINKER_TARGET = '__AMP_CUSTOM_LINKER_TARGET__';
```

Comment:
Please add a comment to explain that this is used to retarget events originating from shadow DOM